pkg/api/mock/common_test.go

```diff
@@ -12,15 +14,20 @@ type testObject struct {
 }
 
 func (o *testObject) EndpointURL(ctx context.Context) (*url.URL, error) { return nil, nil }
+func (o *testObject) GetIdentifier(context.Context) (string, error)     { return o.Identifier, nil }
 
 type testObject2 struct {
 	Identifier string `anxcloud:"identifier"`
 }
 
 func (o *testObject2) EndpointURL(ctx context.Context) (*url.URL, error) { return nil, nil }
+func (o *testObject2) GetIdentifier(context.Context) (string, error)     { return o.Identifier, nil }
 
 type testObjectWithoutIdentifier struct{}
 
 func (o *testObjectWithoutIdentifier) EndpointURL(ctx context.Context) (*url.URL, error) {
 	return nil, nil
 }
+func (o *testObjectWithoutIdentifier) GetIdentifier(context.Context) (string, error) {
+	return "", types.ErrObjectWithoutIdentifier
+}
```

Comment:
Is this test object still relevant with now having a compile-time check if it is a full Object?

Reply:
nice catch making it a check for failing `GetIdentifier()`!